org.eclipse.acute.SWTBotTests/pom.xml

```diff
@@ -11,6 +11,7 @@
 	<properties>
 		<testSuite>${project.artifactId}</testSuite>
 		<testClass>org.eclipse.acute.SWTBotTests.dotnetnew.AllTests</testClass>
+		<testClass>org.eclipse.acute.SWTBotTests.dotnetrun.AllTests</testClass>
```

Comment:
If this is removed will the dontnetrun tests still run? If not, should I rearrange things so that they can all be called from one class?